Sources/CacheContainer.swift

```diff
@@ -46,3 +46,68 @@ public final class CacheContainer {
         }
     }
 }
+
+final class ThreadSafeDictionary<V: Hashable,T>: Collection {
+    private var dictionary: [V: T]
```

Comment:
Please unify the format of dictionary type declaration.
```suggestion
    private var dictionary: [V : T]
```

Reply:
Fixed by [7c1ad17](https://github.com/cybozu/CachePropertyKit/pull/2/commits/7c1ad1725563f67849a27838b787a38b83b4590c).

Reply:
It looks like it hasn't been fixed.